.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "its/sources/kubernetes/examples"]
 	path = its/sources/kubernetes/examples
 	url = https://github.com/kubernetes/examples
+[submodule "its/sources/kubernetes/helm"]
+	path = its/sources/kubernetes/helm
+	url = https://github.com/helm/examples.git
```

Comment:
Are we ok to depend directly on an external git repository? I think for other analyzers we usually have a fork under SonarSource organization. 
However I see it's already the case for other submodules, so I guess this is intentional.